skills/csharp/experimental/itsmskill/Dialogs/Teams/TicketDialogHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Dialogs.Teams
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AdaptiveCards;
+    using ITSMSkill.Extensions.Teams;
+    using ITSMSkill.Extensions.Teams.TaskModule;
+    using ITSMSkill.Utilities;
+    using Microsoft.Bot.Schema;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Helper class to create adaptive cards
+    /// </summary>
+    public class TicketDialogHelper
+    {
+        public static AdaptiveCard CreateIncidentAdaptiveCard()
+        {
+            try
+            {
+                // Json Card for creating incident
+                // TODO: Replace with Cards.Lg and responses
+                AdaptiveCard adaptiveCard = AdaptiveCardHelper.GetCardFromJson("Dialogs/Teams/Resources/CreateIncident.json");
+
+                adaptiveCard.Actions.Add(new AdaptiveSubmitAction()
+                {
+                    Title = "SubmitIncident",
+                    Data = new AdaptiveCardValue<TaskModuleMetadata>()
+                    {
+                        Data = new TaskModuleMetadata()
+                        {
+                            TaskModuleFlowType = TeamsFlowType.CreateTicket_Form.ToString(),
+                            Submit = true
+                        }
+                    }
+                });
+
+                return adaptiveCard;
+            }
+            catch (AdaptiveSerializationException e)
+            {
+                // handle JSON parsing error
+                // or, re-throw
```

Comment:
should handle the error, ideally log the error to app insights

Reply:
removed throwing the error